artemis/reporting/modules/api_scanner/template_api_vulnerability.jinja2

```diff
@@ -0,0 +1,52 @@
+                            <li>{% trans %}Status Code:{% endtrans %} {{ report.additional_data.status_code }}</li>
+                            <li>
+                                {% trans %}Details:{% endtrans %}
+                                {% set raw_detail = report.additional_data.details %}
+                                {% if "HTTP verb" in raw_detail %}
+                                    {% trans %}Unsupported HTTP Method - The endpoint accepts HTTP methods that should not be allowed, potentially enabling unintended operations.{% endtrans %}
+                                {% elif "vulnerable to SQL Injection" in raw_detail or "vulnerable to SQli" in raw_detail %}
+                                    {% trans %}SQL Injection - Malicious input can manipulate database queries, leading to data leaks or unauthorized access.{% endtrans %}
+                                {% elif "time-based SQL Injection" in raw_detail %}
+                                    {% trans %}Time-based SQL Injection - An attacker can infer data using timing-based payloads, exploiting database behavior.{% endtrans %}
+                                {% elif "OS Command Injection" in raw_detail %}
+                                    {% trans %}OS Command Injection - Arbitrary system commands can be executed on the server, leading to full compromise.{% endtrans %}
+                                {% elif "XSS/HTML Injection" in raw_detail %}
+                                    {% trans %}XSS/HTML Injection - Untrusted input is rendered in the browser, enabling session hijacking or defacement.{% endtrans %}
+                                {% elif "SSTI" in raw_detail %}
+                                    {% trans %}SSTI - Server-Side Template Injection allows attackers to execute code on the server by injecting into templates.{% endtrans %}
+                                {% elif "fails to implement security authentication" in raw_detail %}
+                                    {% trans %}Missing Authentication - Critical endpoints lack proper access control, allowing unauthorized access.{% endtrans %}
+                                {% else %}
+                                    {{ raw_detail }}
+                                {% endif %}
+                            </li>
+                            {% if report.additional_data.data_leak %}
+                                <li>{% trans %}Data leaks:{% endtrans %} {{ report.additional_data.data_leak }}</li>
+                            {% endif %}
+                            {% if report.additional_data.curl_command %}
+                                <li>{% trans %}CURL Command:{% endtrans %} <code>{{ report.additional_data.curl_command }}</code></li>
+                            {% endif %}
+                        </ul>
+                    </li>
+                {% endif %}
+            {% endfor %}
+        </ul>
+        <p>
```

Comment:
I don't think this paragraph adds anything, it's too generic - I think the per-type descriptions are enough